connectors/sources/google_cloud_storage.py

```diff
@@ -344,6 +386,7 @@ async def get_docs(self, filtering=None):
         Yields:
             dictionary: Documents from Google Cloud Storage.
         """
+        self._initialize_configurations()
```

Comment:
Why do we initialize configurations at two places? Shouldn't we do it at one place, i.e. the constructor?